### DISCUSSION TITLE
Task comments/notes

### DISCUSSION THREAD

**Discussion by filcuk** [almarklein/timetagger#369] [Ideas]
Hi, I wanted to suggest the option of adding comments/notes to tasks.

**How**
The current `Description` serves as a title.
I was thinking an additional, multi-line textbox that would either show on task edit, in exports and possibly tooltips.

**Why**
I would sometimes like to take notes for the job I'm working on, that would help me to adjust the tags later or for when I'm transferring data to the company time tracking system, needing additional details.

**How would this work on continued tasks**
Keeping it simple and only on the task enterred would be quite sufficient.
Maintaining a comment thread across tags would be interesting, but (I imagine) hard to implement and possibly unecessary.

As aslways, your consideration is much appreciated.

**Comment by almarklein** [comment]
Hi, thanks for the feedback!

I made an issue to track this idea:  #375. In short, I've listed it as a possible feature improvement, but it would mean a change in design and in how the app would potentially be used, so it'd need more thought.
